src/server/graphql/models/User/userMutation.js

```diff
@@ -151,7 +151,13 @@ export default {
         createdAt: new Date(userInfo.created_at),
         tms: userInfo.tms
       };
-      const {email, id: userId, picture, preferredName} = auth0User;
+      const {email, id: userId, createdAt, picture, preferredName} = auth0User;
```

Comment:
Agreed! So that means we only need to call it when the account is created.
Per the docs:
> We recommend calling identify a single time when the user’s account is first created, and only identifying again later when their traits change. - https://segment.com/docs/sources/server/node/

Traits are initialized on signup, but are never changed on signin.